backend/app/api/v1/lantern_api.py

```
@@ -1,12 +1,13 @@
 from typing import Optional, List
 
-from fastapi import APIRouter, Depends, UploadFile, File, Form, Query
+from fastapi import APIRouter, Depends, UploadFile, File, Form, Query, Path
 
 from app.core.db.database import get_mongo_client
 from app.core.exceptions.types import ValidationError
 from app.core.response.response import success_response, error_response
```

Flake8 (7.2.0)

[error] 'app.core.response.response.error_response' imported but unused

(F401)